src/bpmncwpverify/bpmn.py

```diff
@@ -114,6 +115,12 @@ def accept(self, visitor: "BpmnVisitor") -> None:
         self.traverse_outflows_if_result(visitor, result)
         visitor.end_visit_task(self)
 
+    def cleanup(self) -> None:
```

Comment:
Wasn't this function going away?

Reply:
yes, I must have missed it while solving merge conflict, I will take care of this.